apps/admin-gui/src/app/vos/pages/group-detail-page/group-applications/group-applications.component.html

```diff
@@ -12,18 +12,17 @@ <h1 class="page-subtitle">{{'VO_DETAIL.APPLICATION.TITLE' | translate}}</h1>
   <perun-web-apps-debounce-filter
     (filter)="applyFilter($event)"
     [placeholder]="'VO_DETAIL.APPLICATION.FILTER'"></perun-web-apps-debounce-filter>
-  <perun-web-apps-date-range [endDate]="endDate" [startDate]="startDate">
+  <perun-web-apps-date-range [endDate]="endDate" [startDate]="startDate" class="mr-2">
```

Comment:
Bootsrap 5 - margin right -> margin end
```suggestion
  <perun-web-apps-date-range [endDate]="endDate" [startDate]="startDate" class="me-2">
```